engine/client/voice.c

```diff
@@ -0,0 +1,368 @@
+#include "voice.h"
+
+wavdata_t *input_file;
+fs_offset_t input_pos;
+
+voice_state_t voice;
+
+CVAR_DEFINE_AUTO( voice_enable, "1", FCVAR_ARCHIVE, "enable voice chat" );
```

Comment:
This and other client-specific cvars probably should FCVAR_PRIVILEGED, so server will not change them arbitrarily.